CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# Unreleased
+
+## Fixed
+
+- All of the variables can now be overridden by users
+- Sizing of checkboxes & radios now works by setting the `$bitstyles-checkbox-size` and `$bitstyles-radio-size` variables respectively
```

Comment:
Is this "breaking" 👀 ?

Reply:
Ahh, sorry this is something better factored out into its own PR — I‘ll do that!